### PR TITLE
 Introduce the Training manual in Getting started chapter 

### PR DESCRIPTION
**Pull request by DelazJ**
and add some illustrations to getting started sample section (refs #3922 - I'd have loved to add more screenshots but seems there's a [bug with the label rendering](https://github.com/qgis/QGIS/issues/30583), hence screenshots would have not matched the text)

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/user_manual/introduction/getting_started.rst [hunk 1]
```diff
@@ -227,6 +263,11 @@ QGIS, configure them and generate your map in an image format you can use in
 other softwares. Let's move on to learn more about the available functionality,
 features and settings, and how to use them.
 
+.. note::
+ If you want to continue learning QGIS through the step-by-step exercises
+ route, QGIS documentation provides a :ref:`Training manual
+ <QGIS-training-manual-index-reference>` you can rely on.
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]: Can this be simplified to, for instance:
```
To continue learning QGIS through step-by-step exercises, try the :ref:`Training manual
 <QGIS-training-manual-index-reference>`.
```

Comment (DelazJ) [reply source/docs/user_manual/introduction/getting_started.rst]: Can we find a more enforcing word than `try`? The "training manual" should be advertised firmly IMHO and you are never better served than by yourself :smile:

Comment (havatv) [reply source/docs/user_manual/introduction/getting_started.rst]: I have adjusted the suggestion - is it better?

Comment (DelazJ) [reply source/docs/user_manual/introduction/getting_started.rst]: How about `follow the tm` (to keep a simplified sentence as suggested firstly)?

Comment (havatv) [reply source/docs/user_manual/introduction/getting_started.rst]: I am not sure if "follow the <tm>" is proper English, but I let us give it a try... :-)

Comment (DelazJ) [reply source/docs/user_manual/introduction/getting_started.rst]: Let's wait for a day if someone comes with proper English...

**Pull request merged**